FirebaseAppCheck/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+#Unreleased
+- [feature] Added limitedUseTokenWithCompletion() for obtaining limited-use tokens for protecting non-Firebase backends.
```

Comment:
This is debatable but maybe we should consider writing the Swift method signature in the changelog (`limitedUseToken(completion:)`) instead of the ObjC one (`limitedUseTokenWithCompletion()`) even though it's implemented in ObjC.

nit: Please add the PR # to the entry.

```suggestion
- [feature] Added `limitedUseToken(completion:)` for obtaining limited-use tokens for
  protecting non-Firebase backends. (#11086)
```

Reply:
Yes, makes sense to put the swift version, thx